web-app/js/portal/utils/ObservableUtils.js

```diff
@@ -0,0 +1,16 @@
+Ext.namespace('Portal.utils');
+
+Portal.utils.ObservableUtils = {
+    FUNCTIONS_TO_FORWARD: ['on', 'un', 'fireEvent'],
```

Comment:
Yeah, I just forwarded the events I needed. An alternative which I only thought of because of your comment, would be to get the function names programmatically from the Observable class so that you overwrote them all and didn't need to maintain a list of them all.